.vscode/settings.json

```diff
@@ -18,5 +18,6 @@
   },
   "extensions.ignoreRecommendations": true,
   "files.autoSave": "off",
-  "debug.allowBreakpointsEverywhere": true
+  "debug.allowBreakpointsEverywhere": true,
+  "gitdoc.enabled": false
```

Comment:
I see that you disabled gitdoc in the project settings. I'm not sure what gitdoc is (maybe it's this VSCode extension?), but it isn't something that we have as a default in the project so I don't think we need a project-level setting to disable it. Should this be a user setting instead?

Reply:
I think I do have this disabled in my user settings, but it automatically adds that every time I open the code unless I globally disable the extension (i.e. disable from the extensions menu, not just setting `gitdoc.enabled` to false. I'll check on that further